config/samples/backends/bases/lvm/lvm.yaml

```diff
@@ -3,6 +3,7 @@ kind: MachineConfig
 metadata:
   labels:
     machineconfiguration.openshift.io/role: master
+    # kubernetes.io/hostname: "<hostname>"
```

Comment:
If I understand the situation correctly, this might be a good place to add a comment, perhaps something like this:

```
For single node OCP, set the machineconfiguration.openshift.io/role to master. For 3-node OCP, change the role from master to worker, and specify the kubernetes.io/hostname  where you want the LVM backend data
to reside.
```
 
Alternatively, would it be easier to add a requirement to add a label (maybe "cinder-lvm") to an OCP node (regardless of single vs. 3-node OCP), and reference the label in the nodeSelector?

Reply:
Ah, thanks for clarifying. So you only use "workers" when there are >3 nodes.